app/components/hint.tsx

```diff
@@ -198,152 +181,163 @@ export const LoadingOverlay = () => {
-          my={2}
-          mx={2}
-          fontSize={theme.typography.body2.fontSize}
-        >
+        <Typography variant="body3" component="div" mt={2}>
           {error.message}
-        </Box>
+        </Typography>
       ) : null}
-    </Alert>
+    </HintRed>
+  );
+};
+
+export const OnlyNegativeDataHint = () => {
+  return (
+    <HintOrange>
+      <AlertTitle>
+        <Trans id="hint.only.negative.data.title">Negative Values</Trans>
+      </AlertTitle>
+      <Trans id="hint.only.negative.data.message">
+        Negative data values cannot be displayed with this chart type.
+      </Trans>
+    </HintOrange>
   );
 };
 
-export const OnlyNegativeDataHint = () => (
-  <Alert severity="warning">
-    <AlertTitle>
-      <Trans id="hint.only.negative.data.title">Negative Values</Trans>
-    </AlertTitle>
-    <Trans id="hint.only.negative.data.message">
-      Negative data values cannot be displayed with this chart type.
-    </Trans>
-  </Alert>
-);
+export const PublishSuccess = () => {
+  return (
+    <HintGreen>
+      <Trans id="hint.publication.success">
+        Your visualization is now published. You can share and embed it using
+        the URL or the options below.
+      </Trans>
+    </HintGreen>
+  );
+};
 
-export const PublishSuccess = () => (
-  <Alert severity="success">
-    <Trans id="hint.publication.success">
-      Your visualization is now published. You can share and embed it using the
-      URL or the options below.
-    </Trans>
-  </Alert>
-);
+const colorToIcon: Record<NonNullable<AlertProps["color"]>, IconName> = {
+  red: "infoCircle",
+  green: "checkmarkCircle",
+  blue: "infoCircle",
+  orange: "infoCircle",
+};
 
-const mkHint = (severity: AlertProps["severity"], displayName: string) => {
+const mkHint = (
+  color: NonNullable<AlertProps["color"]>,
+  displayName: string
+) => {
   const Component = ({
-    iconName,
-    children,
-    iconSize = 24,
+    smaller,
+    sx,
+    ...rest
   }: {
-    iconName?: IconName;
-    children: ReactNode;
-    iconSize?: number;
-  }) => (
+    smaller?: boolean;
+  } & Omit<AlertProps, "color" | "icon">) => (
     <Alert
-      severity={severity}
-      icon={iconName ? <Icon name={iconName} size={iconSize} /> : false}
-    >
-      {children}
-    </Alert>
+      color={color}
+      icon={smaller ? false : <Icon name={colorToIcon[color]} />}
+      sx={
+        smaller
+          ? { p: 0, backgroundColor: "transparent", boxShadow: 0, ...sx }
+          : sx
+      }
+      {...rest}
+    />
   );
   Component.displayName = displayName;
+
   return Component;
 };
 
-export const HintRed = mkHint("error", "HintRed");
-export const HintYellow = mkHint("warning", "HintYellow");
-export const HintBlue = mkHint("info", "HintBlue");
+export const HintRed = mkHint("red", "HintRed");
+export const HintGreen = mkHint("green", "HintGreen");
+export const HintBlue = mkHint("blue", "HintBlue");
+export const HintOrange = mkHint("orange", "HintOrange");
```

Comment:
Nit: Could be good to have comments here on when to use which. Comments that would appear when hovering the component.